### DISCUSSION TITLE
How can I evaluate a trained agent in a deterministic way?

### DISCUSSION THREAD

**Discussion by fatcatZF** [Toni-SM/skrl#236] [Q&A] [answered]
How can I evaluate a trained agent in a deterministic way like the[ evaluate_policy](https://stable-baselines3.readthedocs.io/en/master/common/evaluation.html) function in stable baselines3?

**Comment by Toni-SM** [comment]
Hi @fatcatZF 

For it, there will be necessary to modify **skrl** trainer's `eval` method or agent's `act` method to get/return the `mean_actions` in a similar way as discussed in https://github.com/isaac-sim/IsaacLab/issues/1078#issuecomment-2383198200

**Comment by Toni-SM** [comment, marked as answer]
Feature added in version 1.4.0: https://github.com/Toni-SM/skrl/blob/d57c8ea138e3356396a8da368f37430f75fb524a/CHANGELOG.md?plain=1#L13-L15
